test/file.e2e-spec.ts

```diff
@@ -490,6 +491,24 @@ describe('File e2e', () => {
         )
         .run();
       await expectConsistency(FileNodeType.File, false);
+      // revert the changes so consistency check will be passed for remaining file nodes.
+      await app
```

Comment:
Since this is clean up, it should be in an `afterEach/afterAll` method so it gets ran regardless of the `expectConsistency` result.